python/lsst/ip/diffim/imagePsfMatch.py

```diff
@@ -423,14 +425,16 @@ def matchExposures(self, templateExposure, scienceExposure,
             if not templateExposure.hasPsf():
                 self.log.warning("No estimate of Psf FWHM for template image")
             else:
-                templateFwhmPix = self.getFwhmPix(templateExposure.getPsf())
+                templateFwhmPix = self.getFwhmPix(templateExposure.getPsf(),
+                                                  templateExposure.getBBox().getCenter())
```

Comment:
I think here and below should explicitly say `position=`

Reply:
Sure, but do you agree that that this is the appropriate position to use?

Reply:
Looking at the code, I believe that this is run _after_ the warping, so that the science and template bboxes should match.  In this case we want to evaluate the psf at the same position (center of bbox) rather than at different positions (individual psf average positions).  So I think this is correct.

Reply:
I set the CoaddPsf's average position to be the *center of mass* of the template. 
This is not the same as the center of the bbox! 
Please keep the existing behavior by evaluating at `psf.getAveragePosition()`

Reply:
I removed the comment referring to DM-32756 if this was intentional.

Reply:
The changes in this snippet breaks ci_hsc, so I removed it. It's an outdated change.